docs/non_containerized_NSFS.md

```diff
@@ -467,13 +467,32 @@ sudo node src/cmd/manage_nsfs bucket delete --config_root ../standalon/config_ro
 sudo node src/cmd/manage_nsfs bucket list --config_root ../standalon/config_root 2>/dev/null
 
 ```
+NSFS management CLI command will create both account and bucket dir if it's missing in the config_root path.
 **Important**:  All the Account/Bucket commands end with `2>/dev/null` to make sure there are no unwanted logs.
 
-
+Using `from_file` flag:
+- Users can also pass account or bucket values in JSON file (hereinafter referred to as "options JSON file") instead of passing them in CLI as arguments using flags.
```

Comment:
please mention that it's for create/update operations

Reply:
added, thanks.